package.json

```diff
@@ -16,7 +16,7 @@
     "link-packages": "./scripts/link-packages.sh",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
-    "lint:changelogs": "cross-env npm_package_repository_url=https://github.com/MetaMask/snaps-skunkworks.git yarn workspaces foreach --parallel --topological run lint:changelog",
+    "lint:changelogs": "yarn workspaces foreach --parallel run lint:changelog",
```

Comment:
I removed `--topological` as well from this command, since the order in which it's ran doesn't matter in this situation.